changelogs/head.asciidoc

```diff
@@ -22,3 +22,4 @@ https://github.com/elastic/apm-server/compare/7.10\...master[View commits]
 * Log warnings in aggregation of transaction metrics when grouping limit is reached {pull}4313[4313]
 * Configurable tail-based sampling policies {pull}4320[4320]
 * Monitoring and telemetry for tail-based sampling {pull}4346[4346] {pull}4360[4360]
+* Support for data streams {pull}4409[4409]
```

Comment:
should we put a note that this is experimental (or is it experimental?)

Reply:
Yep, reworded to "Experimental support"